gradle/verification-metadata.xml

```diff
@@ -5524,7 +5524,9 @@
             <sha256 value="c0b50e68afea3114b04a7443d3d718b9fdedf207dbc7b09702bccb536d6e2956" origin="Generated by Gradle"/>
```

Comment:
sha256 for arithmetic-0.8.4.jar seems incorrect. I get this for the one I downloaded `6bb9bf0bf807e959c83fdcf12fc2187d1fad70de2456fc803593f7c51335cd38`

Reply:
updated with the version generated with a cleared gradle cache